src/app/child-dev-project/reporting/reporting.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { ReportingComponent } from "./reporting/reporting.component";
+import { MatButtonModule } from "@angular/material/button";
+import { MatListModule } from "@angular/material/list";
+import { MatTableModule } from "@angular/material/table";
+import { MatExpansionModule } from "@angular/material/expansion";
+import { MatStepperModule } from "@angular/material/stepper";
+import { MatIconModule } from "@angular/material/icon";
+import { MatDatepickerModule } from "@angular/material/datepicker";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { FormsModule } from "@angular/forms";
+import { AdminModule } from "../../core/admin/admin.module";
+import { ReportRowComponent } from "./reporting/report-row/report-row.component";
+
+@NgModule({
+  declarations: [ReportingComponent, ReportRowComponent],
```

Comment:
shall we move this module into the `app/features/*` folder - or keep it as a part of the core framework?

Reply:
I dont consider this a core functionality because no other module is dependent on this feature. Therefore the features folder seams appropriate